caiman/gui/gui_pyqtgraph_layout.py

```diff
@@ -538,13 +647,21 @@ def change(param, changes):
     
     estimates.filter_components(mov, params_obj, dview=None, 
                                     select_mode=pars_action.param('View components').value())
+    
+    print(estimates.idx_components.shape)
+    
     if mode is "background":
         return
     else:
         draw_contours()                
     
 pars.sigTreeStateChanged.connect(change)
 
+def compute_residual_mov():
+    global bg, mov, mov_residual, estimates
+    reconstructed = (np.array(np.dot(estimates.A.todense(), estimates.C))).reshape([estimates.dims[0],estimates.dims[1],-1], order='F').transpose([2,0,1])
+    mov_residual = mov - bg - reconstructed
```

Comment:
again, you can create all these movies, only for the required pixels and not the entire FOV.